openpype/pipeline/colorspace.py

```diff
@@ -0,0 +1,116 @@
+import re
+import os
+import platform
+import PyOpenColorIO as ocio
+from openpype.settings import get_project_settings
+from openpype.pipeline import legacy_io
+from openpype.lib.log import Logger
+
+log = Logger.get_logger(__name__)
+
+
+def get_imagio_colorspace_from_filepath(
```

Comment:
Typo
```suggestion
def get_imageio_colorspace_from_filepath(
```

Reply:
fixed in https://github.com/pypeclub/OpenPype/pull/4270